app/helpers/check_answers_helper.rb

```diff
@@ -1,154 +1,255 @@
 module CheckAnswersHelper
-  SKIPPABLE_QUESTIONS = %w[
-    rooms_number
-    accommodation_cost
-    transport_type
-    transport_cost
-    offer_space_type
-    space_cost
-    offer_care_qualifications
-    care_cost
-    offer_staff_type
-    offer_staff_charge
-    construction_services
-    construction_services_other
-    construction_cost
-    it_services
-    it_services_other
-    it_cost
-  ].freeze
-
-  def items
-    items = questions.map do |question|
-      # We have answers as strings and hashes. The hashes need a little more
-      # work to make them readable.
-
-      next if skip_question?(question)
-
-      next transport_type if question.eql?("transport_type")
-      next offer_care_qualifications if question.eql?("offer_care_qualifications")
-      next offer_staff_type if question.eql?("offer_staff_type")
-      next link_to_parent_page(question, "offer_staff_type") if question.eql?("offer_staff_charge")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_services_other")
-      next link_to_parent_page(question, "construction_services") if question.eql?("construction_cost")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_services_other")
-      next link_to_parent_page(question, "it_services") if question.eql?("it_cost")
-      next link_to_parent_page(question, "transport_type") if question.eql?("transport_cost")
-      next link_to_parent_page(question, "offer_care_qualifications") if question.eql?("care_cost")
-      next link_to_parent_page(question, "rooms_number") if question.eql?("accommodation_cost")
-      next link_to_parent_page(question, "offer_space_type") if question.eql?("space_cost")
-
-      value = concat_answer(session[question], question)
+  def sections
+    {
+      1 => "accommodation",
+      2 => "transport",
+      3 => "space",
+      4 => "staff",
+      5 => "care",
+      6 => "services",
+      7 => "other_support",
+      8 => "location",
+      9 => "business_details",
+      10 => "contact_details",
+    }.sort.to_h.values
```

Comment:
In Ruby (I think post version 2.1) hashes and arrays are ordered, so they return in the same order consistently. Also the `sort` is acting on the keys so sorting doesn't change the order. That means that the content before the `.sort.to_h` is exactly the same as afterwards. Also I'm not sure what value the numbers provided. I'd suggest simplifying to

```ruby
def sections
  %w[
    accommodation
    transport
    space
    staff
    care
    services
    other_support
    location
    business_details
    contact_details
  ]
end
```

Reply:
Thanks @reggieb 👍  Happy to revert this change back to `%w[...]` as you suggest.  However, the suggestion that caused us to change this was that using an array did not sufficiently indicate that we are expecting an explicit - and predictable - order.  Personally, I feel that - given that code is documentation - the `{ 1 => '', 2 =>, ... x => }sort.to_h.values` approach (whilst redundant as you say) does precisely describe our intent and the desired behaviour.

Reply:
I think the main original criticism of the array was that it was all on one line. Making it multi-line makes it easier to see what the order is and manipulate it as needed. With the numbered array, if you want to change the order, you'll need to move the elements and update all the numbers. So at least twice the complexity.

Personally - of Leena's options, I think the comment would be my favourite.

Reply:
Thanks @leenagupte, @reggieb - I think we're going to have to agree to disagree on this one and I'm clearly out-gunned.  So, I've updated as requested and added a commit comment.